src/pwncore/routes/ctf/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 from fastapi import APIRouter
+from pwncore.routes.ctf import start
```

Comment:
`from pwncore.routes.ctf.start import router as start_router`

because currently it looks like a circular import (less readable)